app/uk/gov/hmrc/contactadvisors/service/EmailService.scala

```diff
@@ -29,17 +29,15 @@ import scala.concurrent.{ ExecutionContext, Future }
 class EmailService @Inject()(emailConnector: EmailConnector, conf: Configuration) {
 
   lazy val clientIds = conf
-    .getOptional[Seq[String]](s"email.clientIds")
-    .getOrElse(List())
-    .toSet
+    .getOptional[Map[String, String]](s"email.clientId")
+    .getOrElse(Map())
 
-  def doSendEmail(payload: JsValue)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Result] =
+  def doSendEmail(payload: JsObject)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Result] =
     if (!validParameters(payload)) {
       Future.successful(BadRequest("""{"error": "invalid parameters"}"""))
```

Comment:
same thing here, do we want a more specific error to tell what parameters are wrong?